CHANGELOG.txt

```diff
@@ -1,15 +1,37 @@
 
 COLMAP 3.11 (11/27/2024)
-------------------------
-Highlights
+========================
+
+New Features
+------------
 * New pose prior based incremental mapper that can leverage absolute pose priors from e.g. GPS measurements.
 * New bundle adjustment covariance estimation functionality. Significantly faster and more robust than Ceres.
 * API documentation with auto-generated stubs for pycolmap.
 * Use PoseLib's minimal solvers for faster performance and improved robustness.
 * Experimental support for CUDA-based bundle adjustment through Ceres (disabled by default).
-* Many more performance optimizations, bug fixes, and other improvements. See full list of changes below.
+* Support for reading 16-bit PNG grayscale images.
+* New RAD_TAN_THIN_PRISM_FISHEYE camera model in support of Meta's Project Aria devices.
+* Replace numerical with analytical Jacobian in image undistortion for better convergence.
+* Many more performance optimizations and other improvements. See full list of changes below.
+
+Bug Fixes
+---------
+* Fixed non-deterministic behavior of CUDA SIFT feature extractor. Broken since 3.10 release.
+* Fixed orientation detection of covariant/affine SIFT feature extractor. Broken since initial release.
+* Fixed point triangulator crashing due to bug in copy of reconstruction. Broken since 3.10 release.
```

Comment:
Triangulation is fixed by https://github.com/colmap/colmap/pull/2704. I think the reconstruction copying issue is for undistortion.

Reply:
Thanks. Fixed.